src/eth/execute.json

```diff
@@ -24,7 +24,6 @@
 		"params": [
 			{
 				"name": "Transaction",
-				"required": true,
```

Comment:
I don't think this modifies the requires of ref'd schemas. I've assembled this local (with `npm run build`) and the output `openrpc.json` still has the same requires. It's possible this is due to the dereffing we do in the build script, but would first check the open rpc spec.

Reply:
I saw that the required fields come from the `Transaction` model, so I removed the required fields from there. Do we also want to make the whole Transaction object not required for `eth_estimateGas`?

Reply:
The issue is that other methods use the same object via reference. So if you change which values are required it will change for all the methods. For this reason, I think we may need to take a step back and more explicitly define the inputs/outputs for each method without worrying as much about code duplication.

Reply:
I created new Optional models then, is that a directions we want to go?